src/chttpd/src/chttpd_util.erl

```diff
@@ -100,3 +101,12 @@ handle_legacy_config(OriginalHeaders, DefaultHeaderValue) ->
         false ->
             OriginalHeaders
     end.
+
+get_db_info_helper(DbName) ->
```

Comment:
We are already in the chttpd_util module which is a "helper" module, so "helper" suffix seems redundant a bit.

Also, other helper functions (get_chttpd_config, etc) don't have a "helper" suffix so this one would stand out as a bit odd